boards/xtensa/esp32_ethernet_kit/board_init.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021 Espressif Systems (Shanghai) Co., Ltd.
+ * Copyright (c) 2022 Grant Ramsay <grant.ramsay@hotmail.com>
```

Comment:
Doing this here looks a bit weird, I mean that if your original code is copied from Expressif, then you should not remove their copyright. If you by accident had the wrong copyright, please fix the sources in previous commits.